.github/workflows/smoke-tests.yml

```diff
@@ -0,0 +1,52 @@
+name: Smoke tests
+on: ['push', 'pull_request']
+env:
+  JHI_SCRIPTS: ./test-integration/scripts
+  JHI_VERSION: 6.9.0
+jobs:
+  applications:
+    name: ${{ matrix.app }}
+    runs-on: ${{ matrix.os }}
+    timeout-minutes: 20
+    strategy:
+      fail-fast: false
+      matrix:
+        node_version: [ 14.15.0 ]
+        os: [ ubuntu-20.04 ]
+        app:
+          - maven-default
+          - maven-postgres-redis
+          - gradle-default
+          - gradle-oracle-hazelcast
+        include:
+          - app: maven-default
+          - app: maven-postgres-redis
+          - app: gradle-default
+          - app: gradle-oracle-hazelcast
```

Comment:
we should be able to remove the include part since they have the same values, don't we?

Reply:
I guess yes, Include could be used to run the same app with different options (for example enable e2e or not) but we don't have it here. I will try to remove this.